examples/sdk-examples/src/examples/geoPushpin/GeoPushpinChartClusteringExample.tsx

```diff
@@ -1,49 +1,47 @@
 // (C) 2020-2022 GoodData Corporation
-import React, { Component } from "react";
+import React from "react";
 import { GeoPushpinChart } from "@gooddata/sdk-ui-geo";
 
 import "@gooddata/sdk-ui-geo/styles/css/main.css";
 
 import { MAPBOX_TOKEN } from "../../constants/fixtures";
 import * as Md from "../../md/full";
 
-export class GeoPushpinChartClusteringExample extends Component {
-    public render() {
-        return (
-            <div style={{ height: "500px", position: "relative" }} className="s-geo-pushpin-chart-clustering">
-                <GeoPushpinChart
-                    location={Md.City.Location}
-                    config={{
-                        mapboxToken: MAPBOX_TOKEN,
-                    }}
-                    onZoomChanged={this.onZoomChanged}
-                    onCenterPositionChanged={this.onCenterPositionChanged}
-                    onLoadingChanged={this.onLoadingChanged}
-                    onError={this.onError}
-                />
-            </div>
-        );
-    }
-
-    private onLoadingChanged(...params: any[]) {
+export const GeoPushpinChartClusteringExample: React.FC = () => {
+    const onLoadingChanged = (...params: any[]) => {
         // eslint-disable-next-line no-console
         return console.log("GeoPushpinChartClusteringExample onLoadingChanged", ...params);
-    }
+    };
 
-    private onError(...params: any[]) {
+    const onError = (...params: any[]) => {
```

Comment:
These `(...params: any[])` can often be replaced by something more approripate. For example this should be

```
const onError: OnError = (error) => ...
```

Please revise this pattern in all the examples and try making the types better (you can check the props these are passed to, they have the correct types).